docs/api/net-log.md

```diff
@@ -36,6 +35,16 @@ Starts recording network events to `path`.
 
 Stops recording network events. If not called, net logging will automatically end when app quits.
 
+**[Deprecated Soon](promisification.md)**
+
+### `netLog.stopLogging()`
+
+* `callback` Function (optional)
```

Comment:
The promisified function doesn't accept a callback does it?

Reply:
@jkleinsc missed that, removed.